include/ismrmrd/serialization.h

```diff
@@ -70,11 +79,34 @@ EXPORTISMRMRD void deserialize(Image<T> &img, ReadableStreamView &rs);
 // deserialize Waveform from istream
 EXPORTISMRMRD void deserialize(Waveform &wfm, ReadableStreamView &rs);
 
+// deserialize const length (1024) char array from istream. Used for CONFIG FILE
+EXPORTISMRMRD void deserialize(char (&str)[1024], ReadableStreamView &rs);
```

Comment:
I think this should explicitly take a "ConfigFile" object, rather than a char array. They are pretty hard to work with in practice, as they decay to char* as soon as they can. 
Alternatively, I would use std::array<char,1024>.

Reply:
Done. I went with ConfigFile in the interface, `std::array` is C++11, which is why I had it as a fixed length char.